src/main/resources/model/instance.json

```diff
@@ -111,6 +111,42 @@
           "showInResponse": true
         }
       }
+    },
+    "classifications": {
+      "type": "object",
+      "properties": {
+        "classificationNumber": {
+          "index": "keyword"
+        }
+      }
+    },
+    "notes": {
+      "type": "object",
+      "properties": {
+        "note": {
+          "index": "multilang"
+        },
+        "staffOnly": {
+          "index": "bool"
+        }
+      }
+    },
+    "electronicAccess": {
```

Comment:
Should we add an 'alias' for all the subfields? So that we will be able to search by `electronicsAccess all "..."`?

In inventory we do support such a search https://github.com/folio-org/mod-inventory-storage/blob/master/src/main/resources/templates/db_scripts/schema.json#L676